src/components/FormInput.tsx

```diff
@@ -1,29 +1,48 @@
 import React, { useState } from 'react';
+import TodoListProps from '../TypescriptProps';
 
 interface FormInputProp {
     addTask: any;
+    listOfItems: TodoListProps[];
 }
 
-const FormInput: React.FC<FormInputProp> = ({ addTask }) => {
-
+const FormInput: React.FC<FormInputProp> = ({ addTask, listOfItems }) => {
     const [ userInput, setUserInput ] = useState('');
-
+    const [ inputError, setInputError ] = useState('');
+    let listCheck = listOfItems.map(listDes => listDes.description);
+    const errorCheck = () => {
+        if (!userInput) {
+            setInputError("No blank spaces");
+            return false;
+        }
+        if (listCheck.includes(userInput.trim())) {
+            setInputError("Item already on list.");
+            return false
+        }
+        return true;
+    }
     const handleChange = (e: { currentTarget: { value: React.SetStateAction<string>; }; }) => {
         setUserInput(e.currentTarget.value)
```

Comment:
so could move this line to the handesubmit function as it would be more performant there?